src/seb/registered_models/sentence_transformer_models.py

```diff
@@ -540,6 +540,25 @@ def create_memo_bert_03() -> SebModel:
     )
 
 
+
+@models.register("nicher92/embedding_model_one_epoch")
```

Comment:
I think the test fails due to the naming here:

```suggestion
@models.register("embedding_model_one_epoch")
```